backend/src/main/java/wooteco/retrospective/application/member/MemberService.java

```diff
@@ -0,0 +1,30 @@
+package wooteco.retrospective.application.member;
+
+import org.springframework.stereotype.Service;
+import wooteco.dto.member.MemberLoginRequest;
+import wooteco.dto.member.MemberLoginResponse;
+import wooteco.exception.member.MemberNotFoundException;
+import wooteco.retrospective.dao.member.MemberDao;
+import wooteco.retrospective.domain.member.Member;
+
+import java.util.Optional;
+
+@Service
+public class MemberService {
+
+    private final MemberDao memberDao;
+
+    public MemberService(MemberDao memberDao) {
+        this.memberDao = memberDao;
+    }
+
+    public MemberLoginResponse loginMember(MemberLoginRequest request) {
+        if (memberDao.exists(request.getName())) {
+            Optional<Member> member = memberDao.findByName(request.getName());
+            return MemberLoginResponse.of(member.orElseThrow(MemberNotFoundException::new));
```

Comment:
```suggestion
            Member member = memberDao.findByName(request.getName()).orElseThrow(MemberNotFoundException::new);
            return MemberLoginResponse.of(member);
```

저는 개인적으로 이렇게 하는게 가독성도 좋고 쓰기도 편하다고 생각해요..! 위에 두 분의견도 맞는 거 같아요..~~~